Sec Hud Nerf

astralenigma opened the pull request:
Just joking this only fixes the permawanted status.
Fixes #8654 
Fixes #8517

astralenigma committed:
Merge remote-tracking branch 'tgstation/master'

astralenigma committed:
Merge remote-tracking branch 'tgstation/master'

astralenigma committed:
Merge remote-tracking branch 'tgstation/master'

astralenigma committed:
This should fix omnivident Sec hud

Fixes #8407
Fixes #8654

astralenigma committed:
I'm feeling jealous about my name not being in there.

astralenigma committed:
Added the hud refresh to the plastic surgery.

astralenigma committed:
Fixes the sec hud now I only need it to work with when a goddamn mask changes.

astralenigma committed:
Moves the hud update code to the update icons instead of the inventory.

PKPenguin321 commented:
nice

KorPhaeron commented:
Finally, thank you.

Supermichael777 commented:
squash those comits

phil235 commented on code/modules/surgery/plastic_surgery.dm:
```diff
@@ -16,9 +16,15 @@
 		user.visible_message("[user] alters [oldname]'s appearance completely, they are now [newname]!", "<span class='notice'>You alter [oldname]'s appearance completely, they are now [newname].</span>")
+		if(ishuman(target))
+			var/mob/living/carbon/human/H = target
+			H.sec_hud_set_ID()
```
you can remove the duplicate code by simply having the three lines once just before the return 1

astralenigma committed:
Made only change the security status on mask and helmet change

ID change only really happens when the ID change either way so it's less
code being processed in 2 places.
Removed copy pasta

astralenigma committed:
I bring back a if to prevent the search of Unknowns in the records and to reduce CPU when Unknowns are under the HUD.

I think this might be my last commit.

Dorsisdwarf commented:
@astralenigma 

Oi. conflicts

phil235 commented:
I made a new version of this PR.
